render.go

```diff
@@ -30,8 +31,12 @@ func New(client client.Renderer, assetsPath, siteDomain string) *Render {
 
 // NewWithDefaultClient returns a render struct with a default rendering client provided (default: unrolled/render)
 func NewWithDefaultClient(assetFn func(name string) ([]byte, error), assetNameFn func() []string, assetsPath, siteDomain string) *Render {
+	isDevelopment := false
+	if strings.Contains(siteDomain, "localhost") {
+		isDevelopment = true
```

Comment:
Can you update the Godoc for this func to state when/how the `isDevelopment` flag is set true - seems like it would be useful for callers of this func to know about.

Reply:
Good idea, thanks. Have updated to highlight how development mode is discerned